pkg/network/netlink/conntrack.go

```diff
@@ -36,6 +36,10 @@ type Conntrack interface {
 // `netNS` is the network namespace for the conntrack operations.
 // A value of `0` will use the current thread's network namespace
 func NewConntrack(netNS netns.NsHandle) (Conntrack, error) {
+	if !isNetlinkConntrackSupported() {
+		return nil, ErrNotPermitted
+	}
```

Comment:
Would it yield better log messages to have `isNetlinkConntrackSupported` be `checkNetlinkConntrackSupported() error` that can return an error that explains if you can't probe capabilities at all vs are missing a capability?